### ISSUE TITLE
[Meru - AB] Layout - New Actions screen

### ISSUE THREAD

**Issue by philipperolet** [dust-tt/dust#3783]
Details on what to do in the **[Figma](https://www.figma.com/file/QOxHwppG64GtPocpDhS6Y7/Product?type=design&node-id=4900-39294&mode=design&t=6tNvt0C7wUTsOHc4-4)** (also, see how it works in the [Prototype](https://www.figma.com/proto/QOxHwppG64GtPocpDhS6Y7/Product?page-id=3366%3A31345&type=design&node-id=4900-40669&viewport=1173%2C2546%2C0.27&t=lV2M2VcNHLWO3Bpe-1&scaling=min-zoom&starting-point-node-id=4900%3A40669) + context & discussion in the [making issue](https://github.com/dust-tt/decisions/issues/164))

- Action selection is now multi-steps (see prototype). First choose action, then if 'use data source' choose how to use datasource (then pick datasource); for dust app just choose the app
- Note the various [action management phases](https://www.figma.com/file/QOxHwppG64GtPocpDhS6Y7/Product?type=design&node-id=4758-31108&mode=design&t=6tNvt0C7wUTsOHc4-4) in the figma
- "Help Me" button is out of scope of this issue, will be done in another one
- New "Select data source" will also be done in another issue

**Comment by philipperolet** [comment]
⚠ **Depends on #3856 ** since in this new screen there is no more slack integration

**Comment by spolu** [comment]
Phase I done here: https://github.com/dust-tt/dust/pull/3920 (bulky code refactor) 

Moving to 2nd Pr that will implement all details of the design
